File: frontend/src/app/models/branch/branch-details-dto.ts
import { UserDto } from '../user/user-dto';

export interface BranchDetailsDto {
    id: number,
    name: string;
    isActive: boolean;
    lastUpdatedBy: UserDto;
    behind: number;
    ahead: number;
    createdAt: Date;
    updatedAt: Date;
}

File: backend/Squirrel.Core/Squirrel.Core.Common/DTO/Branch/BranchDetailsDto.cs
using Squirrel.Core.Common.DTO.Users;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Squirrel.Core.Common.DTO.Branch;
public class BranchDetailsDto: BranchDto
{
    public UserDto? LastUpdatedBy { get; set; } = null!;
    public int Ahead { get; set; }
    public int Behind { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime? UpdatedAt { get; set; } = null!;
}
